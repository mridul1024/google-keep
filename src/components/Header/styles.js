import { makeStyles } from "@material-ui/styles";

export const useStyles = makeStyles({
  header: {
    letterSpacing: "2px",
    fontWeight: "lighter",
    textTransform: "uppercase",
  },
  appBar: {
    background: "#fff59d",
    color: "black",
  },
});
