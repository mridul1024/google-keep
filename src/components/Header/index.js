import React from 'react'
import styled from "styled-components";
import { AppBar, Toolbar, Typography } from "@material-ui/core";
import { useStyles } from './styles';


export default function Header() {
    const classes = useStyles();

    return (
        <>
            <AppBar position="static">
                <Toolbar>
                    <Typography variant="h6" className={classes.header}>
                        Google Keep Clone
                    </Typography>
                </Toolbar>
            </AppBar>
        </>
    )
}
