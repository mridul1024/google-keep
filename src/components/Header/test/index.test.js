import React from "react";
import { render, screen, fireEvent } from "@testing-library/react";
import Header from "../index";

describe("<Header /> component tests", () => {
    test("should render the component and match snapshot", () => {
        let header;
        header = render(<Header />);
        expect(header.baseElement).toMatchSnapshot("baseElement");
    });
});