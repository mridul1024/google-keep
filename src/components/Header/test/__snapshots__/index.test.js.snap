// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header /> component tests should render the component and match snapshot: baseElement 1`] = `
<body>
  <div>
    <div
      class="sc-bdvvtL bMOmCi"
    >
      <p
        class="sc-gsDKAQ iFqnxm"
      >
        Google Keep Clone
      </p>
    </div>
  </div>
</body>
`;
